dags/leanplum_export.py

```diff
@@ -0,0 +1,33 @@
+from airflow import DAG
+from airflow.models import Variable
+from datetime import datetime, timedelta
+from utils import leanplum
+
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2019, 10, 10),
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+
+with DAG('leanplum_export',
+         default_args=default_args,
+         schedule_interval='@daily') as dag:
+
+    app_id = Variable.get('LEANPLUM_FENNEC_NIGHTLY_APP_ID')
+    client_key = Variable.get('LEANPLUM_FENNEC_NIGHTLY_CLIENT_KEY')
+
+    fennec_nightly_export = leanplum.export(
+        task_id='fennec_nightly_export',
+        leanplum_app_id=app_id,
+        leanplum_client_key=client_key,
+        gcs_bucket='moz-fx-data-leanplum-export',
```

Comment:
This bucket doesn't exist. Which bucket it is doesn't matter, we could add a general "external datasets" bucket and have the prefix be leanplum-specific.

Reply:
is there a bug to create the gcs bucket and bq dataset? We would need to know which project, and potentially give the derived dataset svc account access to that bucket if it lives in another project

Reply:
There is not. I don't care necessarily where they are located so long as it is consistent.

Reply:
Bug has been filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1588654